lib/aruba/processes/basic_process.rb

```diff
@@ -115,16 +115,13 @@ def before_run; end
       def after_run; end
 
       def inspect
-        out = stdout(:wait_for_io => 0) + stderr(:wait_for_io => 0)
+        out = truncate("#{stdout(:wait_for_io => 0).inspect}", 35)
+        err = truncate("#{stderr(:wait_for_io => 0).inspect}", 35)
```

Comment:
35 <> 76. Please keep it 76 characters long for now. Strictly speaking this is an API change. Why was that changed? Normally terminals should be 80 characters wide.